vsphere/resource_vsphere_compute_cluster.go

```diff
@@ -1498,15 +1505,17 @@ func resourceVSphereComputeClusterApplyVsanConfig(d *schema.ResourceData, meta i
 	}
 
 	// handle remote datastore/HCI Mesh in a separate call
-	datastoreConfig, err := expandVsanDatastoreConfig(d, meta)
-	if err != nil {
-		return err
-	}
-	if err := vsanclient.Reconfigure(meta.(*Client).vsanClient, cluster.Reference(), vsantypes.VimVsanReconfigSpec{
-		Modify:          true,
-		DatastoreConfig: datastoreConfig,
-	}); err != nil {
-		return fmt.Errorf("cannot apply vsan remote datastores on cluster '%s': %s", d.Get("name").(string), err)
+	if version.AtLeast(viapi.VSphereVersion{Product: version.Product, Major: 7, Minor: 1}) {
```

Comment:
same as above

Reply:
Nice catch - it's 7.0.1 and this would look for 7.1.

Reply:
```suggestion
	if version.AtLeast(viapi.VSphereVersion{Product: version.Product, Major: 7, Minor: 0, Patch: 1}) {
```

Reply:
Suggestion above.